static/css/components/work.less

```
@@ -40,7 +40,7 @@ div.editionAbout {
 
   p {
     margin: 1em 0;
-    font-size: 13px;
+    font-size: @font-size-body-small;
```

Ah not that small :P Medium should do the trick!
```suggestion
    font-size: @font-size-body-medium;
```